pytest_djangoapp/tests/test_fixtures_settings.py

```diff
@@ -90,3 +90,11 @@ def test_context_manager(settings):
 
     settings.OTHER = 1
     assert settings.OTHER == 1
+
+
+def test_urlconf(settings):
+    assert settings.ROOT_URLCONF == 'pytest_djangoapp.tests.testapp.urls'
```

Comment:
This feels very wrong in this case, as the model in this test app is called `testapp.Article` , and it belongs in app `testapp`, and thus the other dotted-modules "should" be relative to the same base directory.

Gut feeling is that it should be `'testapp.urls'`, and `pytest_djangoapp/tests/` added to `PYTHONPATH`.

But, I recognise that urlconf is a project thing, not an app thing, so maybe this is the best solution at the moment.

Reply:
https://github.com/idlesign/pytest-djangoapp/pull/18 removes `ROOT_URLCONF` from the default settings entirely, adding it to settings during the test only when it is needed.

I would not mind if this PR is blocked until it includes a clearer definition of "project", "app", and "testapp" , and has better tests for those.